### PR TITLE
Eng 86

### PR DESCRIPTION
**Pull request by MagdaLis**
Description of scoring and models limitations.

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -35,27 +35,66 @@ Id | Name | Description |
...
+
+<aside class="notice">
+Only textual content in English-language is scored. 
+</aside>
+
+### Scraping
+We scrape and score the main text's body (e.g. an article, blog post, etc) as well as comments posted under it on the same page. The rationale is that if there are any harmful views expressed on a website (irrespectively of whether they are present in the article or in the comments), a reader visiting the site is at a risk of exposure to harmful content. 
+
+### Models
+Factmata's technology is continously improved to provide the best results. We work with domain experts to obtain data to train our models to detect nuanced types of harmful content. Our focus is to ensure results are unbiased and fair. We regularly monitor the performance of our models to identify any issues and resolve them. Below, we list the models' known limitations.
```
Comment (ks2002119) [inline source/index.html.md:77]: our models to identify any issues and resolve them ->  our models to identify issues and resolve them

File: source/index.html.md [hunk 2]
```diff
@@ -35,27 +35,66 @@ Id | Name | Description |
...
+
+Political bias - this algorithm works best on American content, especially relating to 2016 elections. However, it performs less well on political content from other countries.
+
+Hate speech - this model is sensitive to occurence of ethnicity, religious and gender terms, which can lead to false positives.
+
+Sexism - this model is sensitive to occurrence of slurs and words related to sexist abuse, e.g. ‘rape’. The model however does not distinguish between text expressing sexist intent and text reporting sexist views. As a result, the latter can get incorrectly classified as sexist.
+
+Racism - this model works best on detecting islamophobia and antisemitism. Other groups are less well represented and false negatives can occur.
```
Comment (ks2002119) [inline source/index.html.md:85]: islamophobia -> Islamophobia? a few other capitalization too in this section

File: source/index.html.md [hunk 3]
```diff
@@ -35,27 +35,66 @@ Id | Name | Description |
...
+
+Threat - this algorithm detects an intention to inflict damage to an individual or a group. It is a little conservative, leading to some false negatives.
+
+Obscenity - this algorithm detects explicitly profane language. The model can be sensitive to polysemy, i.e. words which have both offensive and non-offensive meaning (e.g. a sport term ‘cockblock’). Without relevant context, this can lead to a false positive.
+
+Clickbait -  clickbait refers to headlines which at the expense of being informative, are designed to entice readers into clicking through. Such titles usually try to exploit the "curiosity gap" by avoiding provision of specific information. Frequently, clickbait titles are framed as questions, e.g. "What is the best diet in the world?" or refer to lists of things without actually naming them, e.g. "23 places you must visit before you die." As a result, the algorithm can be slightly oversensitive to titles with digits or titles framed as questions, resulting in some false positives on such titles.
```
Comment (ks2002119) [inline source/index.html.md:95]: clickbait -> Clickbait

Comment (MagdaLis) [reply source/index.html.md:95]: Approach taken is: 
Caps for model names (Clickbait)
small letters for concepts (clickbait)

**Pull request closed without merging**